pkg/agent/baker.go

```diff
@@ -908,6 +908,14 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"ShouldDisableSSH": func() bool {
 			return config.SSHStatus == datamodel.SSHOff
 		},
+		"GetSysctlContent": func() string {
+			sysctlTemplate := template.Must(template.New("sysctl").Parse(sysctlTemplateString))
+			var b bytes.Buffer
+			if err := sysctlTemplate.Execute(&b, profile); err != nil {
+				panic(fmt.Errorf("failed to execute sysctl template: %s", err))
```

Comment:
is panicking really what we should be doing in this case?

Reply:
I see, I guess we do panic in other spots too

Reply:
yeah good point, I used the commit from previous iteration, let me just return the error.

Reply:
yeah, probably not, updated func to return error